config/samples/lightgbm-dist/README.md

```diff
@@ -0,0 +1,210 @@
+### Distributed Lightgbm Job train
+
+This folder containers Dockerfile and Python scripts to run a distributed Lightgbm training using the XGBoost operator.
+The code is based in this [example](https://github.com/microsoft/LightGBM/tree/master/examples/parallel_learning) in the official github repository of the library.
```

Comment:
Would it be better to point it to LightGBM repo so that we don't have to maintain a copy of the example and dataset?

Reply:
I changed to downloaded the data when building the image instead [commit](https://github.com/kubeflow/xgboost-operator/pull/103/commits/41333ca395b791a6c5814a5a3acf331954760bab)

Reply:
What about the code? Should we use the cloned copy directly instead of copying the code to this repo?

Reply:
The Python code is custom for this example, I am reutilizing the data and the model parameters